MIT.md

```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [YOUR NAME]
```

Comment:
- `[OPTIONAL]` Kindly add your name instead of *[YOUR NAME]* because the license gives express permission for users to reuse code for any purpose, sometimes even if the code is part of proprietary software. A license help user to know who built this software.